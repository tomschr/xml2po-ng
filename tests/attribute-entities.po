msgid ""
msgstr ""
"Project-Id-Version: ентитети\n"
"POT-Creation-Date: 2005-03-27 19:38+0200\n"
"PO-Revision-Date: 2005-03-27 19:38+0200\n"
"Last-Translator: Данило <EMAIL@ADDRESS>\n"
"Language-Team: српски <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: attribute-entities.xml:2 (a)
msgid "test"
msgstr "проба"

#: attribute-entities.xml:2 (p)
msgid "This is just a <placeholder-1/>."
msgstr "Ово је само <placeholder-1/>."

#. Put one translator per line, in the form of NAME <EMAIL>, YEAR1, YEAR2
#: attribute-entities.xml:0 (None)
msgid "translator-credits"
msgstr "Данило"
