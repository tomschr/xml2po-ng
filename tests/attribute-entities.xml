<test>
<p>This is just a <a href="http://example.com/?blah=1&amp;blah=2" xml:lang="sr">test</a>.</p>
</test>
