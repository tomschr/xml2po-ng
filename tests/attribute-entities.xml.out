<?xml version="1.0" encoding="utf-8"?>
<test>
<p>Ово је само <a href="http://example.com/?blah=1&amp;blah=2" xml:lang="sr">проба</a>.</p>
</test>
